docs/integrations/saas-cloud/cisco-amp.md

```diff
@@ -0,0 +1,172 @@
+          "group": "https://api.amp.cisco.com/v1/groups/b077d6bc-bbdf-42f7-8838-a06053fbd98a"
+        }
+      },
+      "file": {
+        "disposition": "Malicious",
+        "file_name": "wsymqyv90.exe",
+        "file_path": "\\\\?\\C:\\Users\\Administrator\\AppData\\Local\\Temp\\OUTLOOK_TEMP\\wsymqyv90.exe",
+        "identity": {
+          "sha256": "b630e72639cc7340620adb0cfc26332ec52fe8867b769695f2d25718d68b1b40",
+          "sha1": "70aef829bec17195e6c8ec0e6cba0ed39f97ba48",
+          "md5": "e2f5dcd966e26d54329e8d79c7201652"
+        },
+        "parent": {
+          "process_id": 4040,
+          "disposition": "Clean",
+          "file_name": "iexplore.exe",
+          "identity": {
+            "sha256": "b4e5c2775de098946b4e11aba138b89d42b88c1dbd4d5ec879ef6919bf018132",
+            "sha1": "8de30174cebc8732f1ba961e7d93fe5549495a80",
+            "md5": "b3581f426dc500a51091cdd5bacf0454"
+          }
+        }
+      },
+      "tactics": [
+        "TA0042"
+      ],
+      "techniques": [
+        "T1204.003"
+      ]
+    }
+  ]
+}
+```
+</details>
+
+### Sample queries
+
+```sql title="Total Events"
+_sourceCategory="Labs/cisco-amp-app"
+| json "id", "connector_guid", "severity", "event_type", "computer.active", "file.disposition", "detection_id", "detection", "computer.hostname", "computer.user", "tactics[*]", "techniques[*]", "computer.external_ip", "file.file_name", "file.file_path", "file.parent.file_name", "file.identity.sha256", "file.identity.sha1", "file.identity.md5", "file.parent.identity.sha256", "date", "computer.network_addresses[*]", "file.parent.process_id", "file.parent.disposition", "computer.links.trajectory", "computer.links.computer", "computer.links.group" as id, connector_guid, severity, event_type, status, file_type, detection_id, detection, hostname, user, tactics, techniques, external_ip, file_name, file_path, parent_file_name, sha2565, sha1, md5, parent_sha256, date, computer_network_addresses, process_id, parent_file_type, trajectory_link, computer_link, group_link nodrop
+
+// global filters
+| where severity matches "{{severity}}"
+| where event_type matches "{{event_type}}"
+| where status matches "{{host_status}}"
+| extract field=tactics "\"?(?<tactics>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| extract field=techniques "\"?(?<techniques>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| where tactics matches "{{tactics}}"
+| where techniques matches "{{techniques}}"
+
+| count by id, connector_guid
+| count
+```
+
+## Set up collection
+
+Follow the instructions provided to set up [Cloud-to-Cloud Integration for Cisco AMP App](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source/) to create the source and use the same source category while installing the app. By following these steps, you can ensure that your Cisco AMP app is properly integrated and configured to collect and analyze your Cisco AMP data.
```

Comment:
```suggestion
To set up [Cloud-to-Cloud Integration Cisco AMP Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source/) for the Cisco AMP app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Cisco AMP Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Cisco AMP app is properly integrated and configured to collect and analyze your Cisco AMP data.
```

Reply:
@amee-sumo Can you update this section with the latest reuse content, because Cisco AMP app too will have source configuration integrated with the app installation.
Ref: https://help.sumologic.com/docs/integrations/saas-cloud/1password/#collection-configuration-and-app-installation